packages/sitecore-jss-dev-tools/src/templating/strip.ts

```diff
@@ -31,15 +27,20 @@ interface StripSettings {
    */
   stripCode?: boolean;
   /**
-   * Pattern to exclude specific directories from iterations
-   * @default /(^|\\)(node_modules|dist|\.next|out|\.generated)(\\|$)/gi
+   * Indicates which files should be included
+   * @default '!(node_modules|dist|.next|out|.generated)/**\/*.@(js|ts?(x))'
```

Comment:
This default comment is incorrect, should be
```suggestion
   * @default '**/*.@(js|ts?(x))'
```

Reply:
Good catch. Done